.github/workflows/node.js.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 15.x]
+        node-version: [14.x, 16.x]
```

Comment:
We can also just add the latest LTS.  
However, I would rather test nodejs `v17` (which is the latest) than `v15`, since the latter is no more supported by nodejs developers (https://endoflife.date/nodejs).

What do you think?

Reply:
Ok, i would keep only the major versions we plan to support for this service